### PR TITLE
[1LP][RFR] Added a noop to shortcut some navmazing fns

### PR DESCRIPTION
**Pull request by psav**
NOOP faster

### CODE REVIEW

**Comment by psav** [comment]
I wanted to keep the fns as fns instead of None

**Comment by RonnyPfannschmidt** [comment]
please rename the `noop` to `can_skip_badness_test`

File: utils/appliance/implementations/ui.py [hunk 1]
```diff
@@ -246,7 +256,7 @@ def check_for_badness(self, fn, _tries, nav_args, *args, **kwargs):
         from cfme import login
 
         try:
-            logger.debug("Invoking {}, with {} and {}".format(fn, args, kwargs))
+            self.log_message("Invoking {}, with {} and {}".format(fn.func_name, args, kwargs))
```
Comment (RonnyPfannschmidt) [inline utils/appliance/implementations/ui.py]: whops, missed before, use `__name__`

**Comment by RonnyPfannschmidt** [comment]
other than the nitpick it looks fine to merge 👍

**Pull request merged**